src/tensor/storage_traits.rs

```diff
@@ -430,15 +430,7 @@ pub trait TensorFrom<Src, S: Shape, E: Unit>: DeviceStorage {
 
 impl<E: Unit, D: DeviceStorage + TensorFromVec<E>> TensorFrom<E, Rank0, E> for D {
     fn try_tensor(&self, src: E) -> Result<Tensor<Rank0, E, Self>, Self::Err> {
-        #[cfg(feature = "no-std")]
-        let buf = {
-            let mut buf = Vec::with_capacity(1);
-            buf.push(src);
-            buf
-        };
-        #[cfg(not(feature = "no-std"))]
-        let buf = vec![src];
-        self.try_tensor_from_vec(buf, ())
+        self.try_tensor_from_vec(vec![src], ())
```

Comment:
I think vec! is only present with std, The old version should work with both?

Reply:
The macro import from `alloc` seems to cover this.

Reply:
Sweet, thanks!